apps/functions/applications-migration/common/migrators/nsip-representation-migration.js

```diff
@@ -59,7 +59,7 @@ export const getRepresentationsForCase = async (log, caseReference) => {
 		return {
 			...row,
 			originalRepresentation: row.originalRepresentation || '',
-			attachmentIds: row.attachmentIds?.split(',') ?? []
+			attachmentIds: row.attachmentIds ? JSON.parse(row.attachmentIds) : []
```

Comment:
Maybe worth exporting `valueToArray` and using it here as well? So in the future if we change this, we can change it everywhere at once.

Reply:
addressed!